src/modules/launcher/Plugins/Microsoft.Plugin.Registry/plugin.json

```diff
@@ -0,0 +1,12 @@
+{
+  "ID": "303417D927BF4C97BCFFC78A123BE0C8",
```

Comment:
You're going to want to add an entry to:: https://github.com/TobiasSekan/PowerToys/blob/RegistryPlugin/.github/actions/spell-check/patterns.txt for this one, e.g.:
```
# plugin.json
^  "ID": "[0-9A-F]{32}",$
```